certified-connectors/TuxMailer/Readme.md

```diff
@@ -0,0 +1,23 @@
+# TuxMailer
+
+TuxMailer is a tool for email validation and marketing services.
+
+## Publisher: Mangesh Sambare
+
+## Prerequisites
+
+No prerequisites
+
+## Supported Operations
+
+### Validate an Email
+
+Validate an email whenever email is added in lists.
+
+## Obtaining Credentials
+
+API Key is required.
+
+## Known Issues and Limitations
+
+No Issues or limitations at the moment of api creation.
```

Comment:
There is no Deployment Instructions section. Please refer to this [file ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)for more information.

Reply:
Deployment Intruction has been added